api/src/main/java/com/redhat/ipaas/api/v1/model/ConnectorGroup.java

```diff
@@ -21,30 +21,30 @@
 import java.io.Serializable;
 
 /**
- * ComponentGroups are labels in Camel.
+ * ConnectorGroups are labels in Camel.
```

Comment:
The IPaaS UI defined Labels as an enduser defined concept. Also they defined a group so a ll the Twitter* Connectors can be mapped to one IPaaS Twitter Connection. We can push back on how the UX team thinks about it but they don't like changing things.. So that said, all the Twitter Connectors in Camel will be labeled with a twitter label, so we can use that to group them in IPaaS. Does that satisfy your concerns, or do you think we should push back to Camel or the UX team?